tests/test_monkeypatch.py

```diff
@@ -1,17 +1,64 @@
+from django.contrib.sites.models import Site
+
+from cms import api
 from cms.cms_toolbars import LANGUAGE_MENU_IDENTIFIER
+from cms.extensions.extension_pool import ExtensionPool
 from cms.test_utils.testcases import CMSTestCase
 from cms.toolbar.toolbar import CMSToolbar
 from cms.toolbar.utils import get_object_edit_url
 from cms.utils.urlutils import admin_reverse
 
 from djangocms_versioning.plugin_rendering import VersionContentRenderer
+from djangocms_versioning.test_utils.extensions.models import (
+    TestPageExtension,
+    TestTitleExtension,
+)
 from djangocms_versioning.test_utils.factories import (
     PageContentFactory,
     PageVersionFactory,
     PollVersionFactory,
+    UserFactory,
 )
 
 
+class MonkeypatchExtensionTestCase(CMSTestCase):
+    def setUp(self):
+        user = UserFactory()
+        self.version = PageVersionFactory(content__language="en")
+        pagecontent = PageContentFactory(
+            page=self.version.content.page, language="de"
+        )
+        self.page = self.version.content.page
+        site = Site.objects.first()
+        self.new_page = self.page.copy(
+            site=site,
+            parent_node=self.page.node.parent,
+            translations=False,
+            permissions=False,
+            extensions=False,
+        )
+
+        new_page_content = api.create_title(
+            page=self.new_page,
+            language=pagecontent.language,
+            slug='slug',
+            path='/path/here',
+            title=pagecontent.title,
+            template=pagecontent.template,
+            created_by=user
+        )
+        self.new_page.title_cache[pagecontent.language] = new_page_content
+
+    def test_copy_extensions(self):
+        '''Try to copy the extension, without the monkeypatch this tests fails'''
+        extension_pool = ExtensionPool()
+        extension_pool.page_extensions = set([TestPageExtension])
+        extension_pool.title_extensions = set([TestTitleExtension])
+        extension_pool.copy_extensions(
+            self.page, self.new_page, languages=['de']
+        )
```

Comment:
No asserts, is this even testing anything? Would it be a pass? I'm not actually sure what would happen, never tried.

Reply:
It does test something. It tests that there are no failures. Removing the monkeypatch and copying the extension fails. There is no self.assertNotRaises, I did add a comment that says that without the monkeypatch it fails.

Reply:
Ok. It should probably explain why it would fail though because if it does start to fail anyone fixing / rewoirkign the test will need to know why the test and the monkey patch are there.

Reply:
Will fix this.

Reply:
```suggestion
        )
        # No asserts, this test originally failed because the versioned manager was called
        # in copy_extensions, now we call the original manager instead
        # https://github.com/divio/djangocms-versioning/pull/201/files#diff-fc33dd7b5aa9b1645545cf48dfc9b4ecR19 
```

Reply:
@Aiky30 Would this comment do?

Reply:
@jonathan-s I can't commit your suggestion as I don't have write access to your branch.